.gitignore

```diff
@@ -1,6 +1,4 @@
-# VS Code
-.vscode

```

Comment:
Please keep `.vscode` ignored since not all users are using `vscode` although I am a fan of vscode.

Reply:
I would still include it, as it will allow to share those settings for people that are using vscode. It doesn't hurt if people are not using .vscode imho.

Reply:
Removed it from this PR. But still think it should be included to get people quickly onboarded.